builtin/logical/transit/path_import.go

```diff
@@ -114,12 +118,27 @@ func (b *backend) pathImportVersion() *framework.Path {
 				Description: `The base64-encoded ciphertext of the keys. The AES key should be encrypted using OAEP 
 with the wrapping key and then concatenated with the import key, wrapped by the AES key.`,
 			},
+			"public_key": {
+				Type:        framework.TypeString,
+				Description: `The plaintext public key to be imported. If "ciphertext" is set, this field is ignored.`,
+			},
 			"hash_function": {
 				Type:    framework.TypeString,
 				Default: "SHA256",
 				Description: `The hash function used as a random oracle in the OAEP wrapping of the user-generated,
 ephemeral AES key. Can be one of "SHA1", "SHA224", "SHA256" (default), "SHA384", or "SHA512"`,
 			},
+			"bump_version": {
+				Type:    framework.TypeBool,
+				Default: true,
```

Comment:
I'm not quite sure I agree with this behavior @stevendpclark  -- I'm not sure this should really be a parameter. Importing a public key always bumps version if it doesn't match the existing private key, importing a private key bumps the version if it doesn't match the public key, and re-importing something matching shouldn't bump the version either. I think there's enough context to do the right thing without adding this parameter IMO.

Reply:
Unless I am not considering or forgetting something, with your suggestion, it seems easy for public keys to be "detached" from their private counterpart, whether on purpose, as other versions are imported on the same key, or by mistake. Just a thought, don't know if would matter or cause any problems.

Reply:
@cipherboy and I discussed this a bit and I believe we came to the conclusion we'd want the behavior he is mentioning but to only operate on the current latest version of the referenced key. That way we can get rid of both new options, the import versions operates more or less the same as it does today in that new imports create a new version except for this one use case in which you happen to match the existing public key and we swap it to a private key. 

Again probably something we will readjust afterwards.